src/Agent.Sdk/Util/PlatformUtil.cs

```diff
@@ -74,6 +74,26 @@ public static bool RunningOnLinux
             get => PlatformUtil.HostOS == PlatformUtil.OS.Linux;
         }
 
+        public static bool RunningOnAlpine
+        {
+            get
+            {
+                try
+                {
+                    if (File.Exists("/etc/alpine-release"))
+                    {
+                        return true;
+                    }
+                }
+                catch (Exception)
```

Comment:
>Exception [](http://example.com/codeflow?start=23&length=9)

Please narrow this to the exception you're expecting here.  If not expecting an exception, remove try/catch block

Reply:
No exceptions are expected here. Fixed, thanks!